relay.module.d.ts

```diff
@@ -17,22 +15,22 @@ export declare const UserIdHex: (userId: Uint8Array) => string;
 
 export declare class Relay {
   public ws: WebSocket;
-  public onClientDisconnect: ((userId: Uint8Array) => void) | null;
+  public onClientDisconnect: ((userId: Uint8Array) => unknown) | null;
   public onSubscription: ((userId: Uint8Array) => Relay) | null;
-  public BinaryMessageHandlers: { [key: keyof re.OP]: (sender: Uint8Array, message: Uint8Array | object) => void };
-  public TextMessageHandlers: { [key: keyof re.OP]: (sender: Uint8Array, message: Uint8Array | object) => void };
-  public VariableCallbacks: Function[];
+  public BinaryMessageHandlers: { [key: number]: (sender: Uint8Array, message: Uint8Array) => unknown };
+  public TextMessageHandlers: { [key: number]: (sender: Uint8Array, message: GenericObject) => unknown };
+  public VariableCallbacks: GenericFunction[];
```

Comment:
Callbacks should have specific function types that match the args that the callbacks would take.  They shouldn't be generic because the types of functions you can pass isn't open-ended, it needs to be a function that takes the correct args.